### PR TITLE
Issue #1332 Updated Document Tooltip Feature in README.md

### LINKED ISSUE #1332: Document Tooltip Feature in README.md
### 💡Format this Issue Into a First Timers Issue

This has been marked as a good candidate for becoming a [first-timers-only issue](https://code.publiclab.org/#r=all) like these, meaning that it's **simple, self-contained**, and with some extra formatting✍️, could be a great entry point for a new contributor. If you're familiar enough with this code, please consider **reformatting it as a first-timers-only issue**, and then ping @publiclab/reviewers to get it labeled🏷. Or, if this is not your first time, **try to solve it yourself**🧑‍💻!

---

Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from underrepresented groups in free and open-source software!

We know that the process of creating a pull request is one of the biggest barriers for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [README.md](https://github.com/publiclab/Leaflet.DistortableImage/blob/main/README.md) in the `Leaflet.DistortableImage` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

Place the diff below in line 80 (i.e., just below bullet point _Suppress toolbar_) and add a single line space afterward.
```diff
+  [tooltipText](#Tooltip-Text)
```

Place the diff below in line 337 (i.e. just above section _UI Functionalities_)
```diff
+  ### Tooltip Text
+  `tooltipText` (*optional*, default: '', value: *string*)
+
+  This provides the flexibility to add tooltip text to every image placed on the tile layer.
+
+  For ex.
+
+  ```js
+  // Sets up tooltip text for an image, the text is displayed when mouse is placed on it
+  img = L.distortableImageOverlay(
+         'example.jpg', 
+         {tooltipText: 'Sample text'}
+  );
+  ```
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

(This issue was created by [First-Timers-Bot](https://github.com/hoodiehq/first-timers-bot).)

### PR DESCRIPTION
**Pull request by sudeep162002**
Fixes #1332

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [✔️ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ✔️] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [ ✔️] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ✔️] screenshots/GIFs are attached 📎 in case of UI updates
* [ ✔️] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by 7malikk** [comment]
Hello @sudeep162002 Great job 🎉 

Kindly add the number to the corresponding issue you solved to help the maintainers. Thank you!

I'm sure it'll get reviewed soon @cesswairimu @TildaDares

**Comment by TildaDares** [comment]
Hi @7malikk, feel feee to approve if you think everything looks good. You can either @cesswairimu or me to get it merged.

Thank you both!

**Comment by 7malikk** [comment]
> Hi @7malikk, feel feee to approve if you think everything looks good. You can either @cesswairimu or me to get it merged.
> 
> Thank you both!

Alright, @TildaDares Thank you!

File: README.md
Comment (7malikk) [inline README.md:80]: Hello @sudeep162002 Could you remove one space before the text
like so `* [tooltipText](#Tooltip-Text)` not `*  [tooltipText](#Tooltip-Text)`
Thank you!

Comment (sudeep162002) [reply README.md]: done

Comment (7malikk) [reply README.md]: Great work @sudeep162002

**Comment by 7malikk** [comment]
@TildaDares, @cesswairimu I believe this is ready for a merge, thank you!

Great job @sudeep162002 🎉

**Comment by 7malikk** [comment]
> @sudeep162002, great effort so far. One little change, please place the second code snippet in line 337 instead (and not 339). Issue #1332 is now updated to reflect this change, sorry for the mix-up.
> 

Thank you for catching that @segun-codes, must've been an oversight on my part.

**Comment by sudeep162002** [comment]
changed the pr according to updated issue please review.

**Comment by segun-codes** [comment]
> Hello @sudeep162002 Great job 🎉
> 
> Kindly add the number to the corresponding issue you solved to help the maintainers. Thank you!
> 
> I'm sure it'll get reviewed soon @cesswairimu @TildaDares

@sudeep162002, please, fix @7malikk's suggestion above (i.e., replace 0000 in **Fixes #0000 (<=== Add issue number here)** with the _issue number_ of the issue this PR is related to).

**Comment by sudeep162002** [comment]
done .

**Comment by segun-codes** [comment]
@7malikk, things look all set here, over back to you.

**Review by TildaDares — APPROVED** [review]
Amazing work @sudeep162002. 

Thank you for the reviews @7malikk @segun-codes!

**Pull request merged**
